includes/compat/class-wc-stripe-compat.php

```diff
@@ -182,17 +268,16 @@ public function process_subscription_payment( $order = '', $amount = 0 ) {
 	 * @param string $stripe_token (default: '')
 	 * @param  bool initial_payment
 	 */
-	public function retry_subscription_payment( $order = '', $amount = 0 ) {
+	public function retry_subscription_payment( $amount = 0, $renewal_order ) {
```

Comment:
`WC_Stripe_Compat::scheduled_subscription_payment` which calls this method says that amount is type of float. In that case, should we have `$amount = 0.0` and update docblock above?